File: generated-sources/php-ze-ph/mojang-sessions/src/App/com/github/asyncmc/mojang/sessions/php/ze/ph/model/PlayerSkinURL.php
<?php

namespace com.github.asyncmc.mojang.sessions.php.ze.ph.server\com.github.asyncmc.mojang.sessions.php.ze.ph.model;

use Articus\DataTransfer\Annotation as DTA;

/**
 * Wraps the URL to the texture and configures the player&#39;s model
 */
class PlayerSkinURL
{
    /**
     * The URL to the texture, must be in Mojang&#39;s domains.
     * @DTA\Data(field="url")
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $url;
}

File: generated-sources/scala-gatling/mojang-sessions/src/gatling/scala/com/github/asyncmc/mojang/sessions/scala/gatling/model/PlayerSkinURL.scala

package com.github.asyncmc.mojang.sessions.scala.gatling.model


case class PlayerSkinURL (
    /* The URL to the texture, must be in Mojang's domains. */
    _url: String
)
object PlayerSkinURL {
    def toStringBody(var_url: Object) =
        s"""
        | {
        | "url":$var_url
        | }
        """.stripMargin
}
